docs/reference/src/language/builtins/globals.md

```diff
@@ -1,6 +1,44 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
 # Builtin Global Singletons
 
+## `Palette`
+
+Use `Palette` to create custom widgets that match the colors of
+the selected style e.g. fluent, cupertino, material, or qt.
+
+### Properties
+
+-   **`background`** (_out_ _brush_): Defines the default background brush. Use this if none of the more specialised background brushes apply.
+-   **`foreground`** (_out_ _brush_): Defines the foreground brush that is used for content that is displayed on `background` brush.
+-   **`alternate-background`** (_out_ _brush_): Defines an alternate background brush.
+-   **`alternate-foreground`** (_out_ _brush_): Defines the foreground brush that is used for content that is displayed on `alternate-background` brush.
+-   **`control-background`** (_out_ _brush_): Defines the default background brush for controls, such as push buttons, combo boxes, etc.
+-   **`control-foreground`** (_out_ _brush_): Defines the foreground brush that is used for content that is displayed on `control-background` brush.
+-   **`accent-background`** (_out_ _brush_): Defines the background brush for highlighted controls such as primary buttons.
+-   **`accent-foreground`** (_out_ _brush_): Defines the foreground brush that is used for content that is displayed on `accent-background` brush.
+-   **`selection`** (_out_ _brush_): Defines the background brush that is used to highlight a selection such as a text selection.
```

Comment:
Oh, I just realize: The proposal we discussed had `selection-background` and `selection-foreground`. Could you briefly comment on why you decided to just go for `selection` instead of two properties?

Reply:
ADDED